RecFCCeeCalorimeter/src/components/CaloTowerToolFCCee.cpp

```diff
@@ -289,6 +296,7 @@ uint CaloTowerToolFCCee::idTheta(float aTheta) const {
 // aPhi Position of the calorimeter cell in phi
 uint CaloTowerToolFCCee::idPhi(float aPhi) const {
   uint id = floor((aPhi + m_phiMax) / m_deltaPhiTower);
+  // uint id = floor(aPhi / m_deltaPhiTower);
```

Comment:
remove this commented line (confusing)
By the way, note for self and @BrieucF - I am puzzled by the way we index the towers... why not  floor((aPhi - m_phiMin) / m_deltaPhiTower);, so that the index will go from 0 to m_nPhiTowers-1? 
With this calculation, the index goes from (phiMin+phiMax)/deltaPhi to (phiMin+phiMax)/deltaPhi + m_nPhiTowers-1, and this is what seems to be assumed by the phiNeighbour method, and the current code only gives the same result by accident (because phiMin=-pi, phiMax=pi and thus phiMin+phiMax = 0 and 2 phiMax = phiMax - phiMin = 2pi)

Reply:
Commented line removed